webapp/channels/src/components/widgets/tag/generic_tag.tsx

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import classNames from 'classnames';
+import React from 'react';
+
+import './generic_tag.scss';
+
+export type GenericTagProps = {
+
+    /**
+     * The text content of the tag
+     */
+    text: string;
+
+    /**
```

Comment:
While a good practice, IMHO these doc strings are unnecessary as what they document is obvious and trivial.

Reply:
agree, comments will be removed in my next commit.